spec/test/fixtures/server_error/tests/server-throw-exception.agent.nut

```diff
@@ -0,0 +1,8 @@
+
+class AgentTestCase extends ImpTestCase {
```

Comment:
Can we please make the tabs consistent across the code.  Let's do 4 spaces per tab everywhere.